cmd/utils.go

```diff
@@ -153,17 +154,32 @@ func commonOwnerOrOperatorSetup(cmd *cobra.Command) (common.Address, *ecdsa.Priv
 	var pk *ecdsa.PrivateKey
 	// if no flag was passed, we just use the operator address by default
 	from := cmd.Flag("from").Value.String()
-	if from == "" {
-		from = opEvm.String()
-		pk, err = ks.GetPrivate(util.OperatorKey)
-	} else if from == opEvm.String() || from == opFevm.String() {
-		pk, err = ks.GetPrivate(util.OperatorKey)
-	} else if from == owEvm.String() || from == owFevm.String() {
+	switch from {
+	case "", opEvm.String(), opFevm.String():
```

Comment:
What happens here if `from` is `""`? Does the following logic still work?

Reply:
Yes, the default was to use the operator if `from` was empty and this maintains that.